site/cypress/e2e/polls.spec.js

```diff
@@ -59,6 +59,14 @@ describe('Test cases revolving around polls functionality', () => {
         cy.contains('Poll 5');
 
         cy.get('#tomorrow-table').contains('Poll 5');
+
+        // delete the poll
+        cy.get('#tomorrow-table-dropdown').click();
+        cy.contains('Poll 5').siblings(':nth-child(2)').click();
+        cy.wait(1000);
```

Comment:
Adding `cy.wait()` calls can result in flaky tests in the CI and should generally be avoided.  Is there another way to do this?